### PR TITLE
Fix refresh-social-networks modal CSS class names

### PR DESCRIPTION
**Pull request by yipingliao**
@2is10 Leo has found a bug where the modal content was hidden by an ad-blocker extension due to clashing CSS class names.

### CODE REVIEW

File: kifi-backend/modules/shoebox/angular/src/social/connectNetworks.styl [hunk 1]
```diff
@@ -1,41 +1,30 @@
...
   font-family: Lato, Arial, sans-serif
   font-style: italic
 
-.social-connect-desc
+.kf-social-connect-desc
   font-size: 16px
   margin: 15px auto 35px
   max-width: 350px
 
-.friends-wrapper, .invite-wrapper
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/social/connectNetworks.styl:18]: From here until line 27, these styles are not being used.

File: kifi-backend/modules/shoebox/angular/src/social/connectNetworks.tpl.html [hunk 2]
```diff
@@ -1,114 +1,114 @@
-<div class="social-connect" ng-show="data.show">
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/social/connectNetworks.tpl.html:1]: Turned out that this `ng-show` is a no-op.

File: kifi-backend/modules/shoebox/angular/src/social/social.js [hunk 3]
```diff
@@ -11,9 +11,6 @@ angular.module('kifi')
       restrict: 'A',
       templateUrl: 'social/connectNetworks.tpl.html',
       link: function (scope/*, element, attrs*/) {
-        scope.data = scope.data || {};
-        scope.data.show = true;
```
Comment (yipingliao) [inline kifi-backend/modules/shoebox/angular/src/social/social.js:15]: Related to previous comment re no-op `ng-show`.

**Comment by 2is10** [comment]
LGTM as long as everything still works. :wink:

**Pull request merged**